### DISCUSSION TITLE
Is there any way to or truncate a float within the Slint language?

### DISCUSSION THREAD

**Discussion by DeonteVanterpool** [slint-ui/slint#1708] [Q&A] [answered]
I want to do math on struct fields and have the UI display the result as an integer rather than the float. Should I just have Rust do the math, or is there a way to truncate within Slint? I realized that creating a new integer property with the expression rounds instead of truncating

**Comment by tronical** [comment, marked as answer]
There's `Math.floor` (and other functions in `Math`), but there's no `trunc` yet. Would that work for you?

**Reply by DeonteVanterpool** [reply]
Yes, thank you
